openid-4-verifiable-presentations-1_0.md

```diff
@@ -549,11 +555,250 @@ The Wallet then validates the request as specified in OAuth 2.0 [@RFC6749].
 
 If the Verifier responds with any HTTP error response, the Wallet MUST terminate the process.
 
+# Verifiable Presentation Query Language {#vp_query}
+
+The Verifiable Presentation Query Language (VP Query) is a JSON-encoded query
+language that allows the Verifier to request the Wallet to present Verifiable
+Credentials that match the query. The query consists of a set of constraints
+that the requested Verifiable Credentials must satisfy. The Wallet evaluates the
+query against the Verifiable Credentials it holds and returns the Verifiable
+Presentations that match the query.
+
+A valid VP Query is defined as a JSON-encoded object with the following
+top-level property:
+
+`expect_credentials`:
+: REQUIRED. An array of Credential Queries that specify the requested
+Verifiable Credentials, as defined in (#credential_query).
+
+Note: While this specification does not define additional top-level properties,
+future extensions MAY define additional properties that modify the behavior of
+the Wallet when processing the query.
+
+## Credential Query {#credential_query}
+
+A Credential Query is an object representing a request for one specific
+credential. It contains the following properties:
+
+`group`:
+: REQUIRED. A string identifying the credential in the response, or, if multiple
+credentials carry the same group identifier, the group of credentials that the
+requested credential belongs to.
+
+`format`:
+: REQUIRED. A string that specifies the format of the requested
+Verifiable Credential. Valid Credential format identifier values are defined in
+Appendix A of [@!OpenID.VCI]. The value of this property MUST be one of the
+supported Credential formats as defined in the Wallet's metadata.
+
+`expect_meta`: 
+: OPTIONAL. An object defining additional properties requested by the Verifier that
+apply to the metadata and validity data of the credential. The properties of
+this object are defined per format in (#format_specific_properties). If omitted,
+no specific restrictions are placed on the metadata or validity of the requested
+Credential.
+
+`expect_claims`:
+: OPTIONAL. An array that specifies the claims that the requested
+Verifiable Credential must contain. Each entry in the array corresponds to exactly
```

Comment:
```suggestion
Verifiable Credential MUST contain. Each entry in the array corresponds to exactly
```

I hope that `requested_claims` sounds better to you too and that we might evaluate this change as well.